pkg/build/api/types.go

```diff
@@ -266,6 +266,18 @@ const (
 	// secret for pushing the output image.
 	// The build will stay in the pending state until the secret is created, or the build times out.
 	StatusReasonMissingPushSecret = "MissingPushSecret"
+
+	// StatusReasonFailedPostCommitHook indicates that the postcommit failed
+	StatusReasonPostCommitHookFailed = "PostCommitHookFailed"
+
+	// StatusReasonGitCloneFailed
+	StatusReasonGitCloneFailed = "GitCloneFailed"
+
+	// StatusReasonPullImageFailed
+	StatusReasonPullImageFailed = "PullImageFailed"
```

Comment:
not to mention @PI-Victor is already using every value in 2 places so far (once when he sets the reason, and a second time when he determines the verbose "message" to set based on the reason value)